sceptre/scipool/templates/sc-product-ec2-linux-docker-notebook.j2

```diff
@@ -0,0 +1,106 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
this file and `sc-product-ec2-linux-jumpcloud-notebook.j2` are nearly identical.  Do we plan to delete  sc-product-ec2-linux-jumpcloud-notebook.j2 after getting this to work?  if so, how about creating a PR for the deletion that's dependent on this PR?

Reply:
Yes, but I think there are other files to delete too. I created a follow on [issue](https://sagebionetworks.jira.com/browse/IT-3394) to remove the original RStudio notebook product:  and listed the files I know about. If you know of other files to be removed, feel free to list them in the issue.

Reply:
i guess issue is fine but preference is to create a PR because it shows intention and work and it's inline with this series of PRs.